### ISSUE TITLE
Does training image location count?

### ISSUE THREAD

**Issue by maurokenny** [ultralytics/yolov5#6824]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hello,

I would like to detect and classify an object but, in all my training data this object is located at the _top_ of my image. Does Yolo take into account the position on an image for the training and verification?

- In other words, if my object is at the _bottom_ of a test image but always at the _top_ on training, how does Yolo treat it? Does it decrease the confidence somehow?

Thanks

**Comment by glenn-jocher** [comment]
@maurokenny CNNs are translationally invariant, so no, location of an object is not relevant.

**Comment by maurokenny** [comment]
Thank you

**Comment by gillonba** [comment]
Different issue, but I had a set of training data where all the training data were squares, and then all the results had the same aspect ratio as the image file being tested.  Weird...
